environments/rinkeby/deploys.yml

```diff
@@ -710,7 +735,8 @@ presale.aragonpm.eth:
       contractAddress: '0x8645E3Cf3d1F1234e6A14ea7e545a51788C9fF31'
       commitHash: 491a4357c51959bc1cc341a612be4fcbcbbe682d
 aragon-fundraising.aragonpm.eth:
-  github: 'https://github.com/aragonblack/fundraising/tree/[ref]/apps/aragon-fundraising'
+  github: >-
```

Comment:
Oh interesting; was this done automatically? Is the `>-` required for a new line?